consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftRound.java

```diff
@@ -84,13 +84,7 @@ public ConsensusRoundIdentifier getRoundIdentifier() {
 
   public void createAndSendProposalMessage(final long headerTimeStampSeconds) {
     final Block block = blockCreator.createBlock(headerTimeStampSeconds);
-    final IbftExtraData extraData = IbftExtraData.decode(block.getHeader().getExtraData());
-    LOG.debug(
-        "Creating proposed block. round={} extraData={} blockHeader={}",
-        roundState.getRoundIdentifier(),
-        extraData,
-        block.getHeader());
-
+    LOG.debug("Creating proposed block. round={}", roundState.getRoundIdentifier());
```

Comment:
nit: tempted to put a trace in here too with the additional info?

Reply:
done. Didn't it was useful at debug level and it made the logs difficult to read. but at trace it's probably appropriate.